zkevm-circuits/src/evm_circuit.rs

```diff
@@ -475,4 +484,81 @@ mod evm_circuit_stats {
             |_, state, _| state.get_step_height_option().unwrap(),
         );
     }
+
+    /// This function prints to stdout a table with the top X ExecutionState
+    /// cell consumers of each EVM Cell type.
+    ///
+    /// Run with:
+    /// `cargo test -p zkevm-circuits --release get_exec_steps_occupancy
+    /// --features test -- --nocapture --ignored`
+    #[ignore]
+    #[test]
+    fn get_exec_steps_occupancy() {
+        let mut meta = ConstraintSystem::<Fr>::default();
+        let circuit = EvmCircuit::configure(&mut meta);
+
+        let report = circuit.0.execution.instrument().clone().analyze();
+        macro_rules! gen_report {
+            ($report:expr, $($id:ident, $cols:expr), +) => {
+                $(
+                let _report = report
```

Comment:
The `_report` variable is used later, why does it have the `_` prefix? Some name suggestions are: "data", "data_matrix", "report_matrix".

Reply:
Ohh just to change the name. As otherwise we override it. Will pick a different name!